external/storm-kafka-client/pom.xml

```diff
@@ -110,7 +110,7 @@
         </dependency>
         <dependency>
             <groupId>org.apache.kafka</groupId>
-            <artifactId>kafka_2.11</artifactId>
+            <artifactId>kafka_2.13</artifactId>
```

Comment:
I'm confused... aren't trying to get version 3.6.0 ?

Reply:
This denotes the scala version, see https://mvnrepository.com/artifact/org.apache.kafka/kafka